lib/doorkeeper/engine.rb

```diff
@@ -5,7 +5,9 @@ class Engine < Rails::Engine
     end
 
     initializer "doorkeeper.locales" do |app|
-      app.config.i18n.fallbacks = [:en]
+      if app.config.i18n.fallbacks.blank?
+        app.config.i18n.fallbacks = [:en]
+      end
     end
```

Comment:
no. 
Because this file is load after app's `application.rb`, when the app set `config.i18n.fallbacks = false`, it will raise `undefined method`.
`config.i18n.fallbacks` has a default value, an empty set, so `app.config.i18n.fallbacks ||= []` has no effect.